src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Basically LGTM, just curious about whether you plan to use/enforce a canonical representation everywhere?

> an example where it would be appropriate to strip?

Well, when in rippled format, it will have the undefined source.amount.counterparty, filled out with Amount.issuer as Account right? And if you want to show transactions sourced from rippled data it in the RippleAPI format that it was declared in, it might make sense to set amount.counterparty to undefined when Amount.issuer == Account?